support-modules/aws/src/main/scala/com/gu/aws/AwsCloudWatchMetricPut.scala

```diff
@@ -45,7 +45,7 @@ object AwsCloudWatchMetricPut {
     metricDatum1.setValue(request.value)
     metricDatum1.setUnit(StandardUnit.Count)
     putMetricDataRequest.getMetricData.add(metricDatum1)
-    Try(client.putMetricData(putMetricDataRequest)).map(_ => Unit)
+    Try(client.putMetricData(putMetricDataRequest)).map(_ => ())
```

Comment:
this was just a bug/nonsense and it doesn't even compile in scala 2.13.  Can put in another PR if necessary but it will need fixing at some point.